api/server/middleware/debug.go

```diff
@@ -27,14 +27,14 @@ func DebugRequestMiddleware(handler func(ctx context.Context, w http.ResponseWri
 			"method":      r.Method,
 			"request-url": r.RequestURI,
 			"vars":        vars,
-			"status":      http.StatusOK,
 		}
+		logger.WithFields(fields).Debugf("handling %s request", r.Method)
 		defer func() {
 			if retErr != nil {
 				fields["error-response"] = retErr
 				fields["status"] = httpstatus.FromError(retErr)
+				logger.WithFields(fields).Debugf("error response for %s request", r.Method)
 			}
-			logger.WithFields(fields).Debugf("handling %s request", r.Method)
```

Comment:
I think it makes sense to uncondtionnally write a debug log when the response is sent, even if everything goes well. That could help detect which request is causing an extra delay on shutdown for instance.

Reply:
Yeah, that's a fair point.

I somewhat tried to avoid spamming too much (e.g. `HEAD /_ping` would produce 2 log entries); wondering if we should have some condition for that :thinking: WDYT?